translations/fr/get5.phrases.txt

```diff
@@ -70,11 +74,11 @@
     }
     "MaxPausesUsedInfoMessage"
     {
-        "fr"            "Votre équipe a déjà utilisé son maximum de {1} pauses."
+        "fr"            "{1} a déjà utilisé son maximum de {2} pauses."
```

Comment:
Pauses at plural, pause in single

Confirm it's use in 1 and more ? If yes, change to pause(s)

Reply:
> Pauses at plural, pause in single
> 
> Confirm it's use in 1 and more ? If yes, change to pause(s)

It can be any number between 1 and whatever, so yes, it should handle both.

Reply:
{1} a déjà utilisé son maximum de {2} pause(s).